test.disabled/ejabberd_tests/tests/gen_mod_SUITE.erl

```diff
@@ -0,0 +1,53 @@
+%%==============================================================================
+%% Copyright 2018 Erlang Solutions Ltd.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%% http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%==============================================================================
+
+-module(gen_mod_SUITE).
```

Comment:
Have you considered making it a small test?

Reply:
No, but indeed it is too small to qualify as a big test :)

Reply:
I'm asking not due to its size but because it's dominated by RPC calls. :) On the other hand it will probably require some mecking so it's up to you if you want to convert it.